data/modules/Equipment/Internal.lua

```diff
@@ -0,0 +1,242 @@
+	icon_name="equip_shield_generator"
+})
+
+Equipment.Register("shield.basic_s5", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=58500, purchasable=true, tech_level=10,
+	slot = { type="shield", size=5 },
+	mass=43.7, volume=78, capabilities = { shield=5 },
+	icon_name="equip_shield_generator"
+})
+
+--===============================================
+-- Hull Modifications
+--===============================================
+
+Equipment.Register("hull.atmospheric_shielding", EquipType.New {
+	l10n_key="ATMOSPHERIC_SHIELDING",
+	price=200, purchasable=true, tech_level=3,
+	slot = { type="hull.atmo_shield", size=1 },
+	mass=1, volume=2, capabilities = { atmo_shield=9 },
+	icon_name="equip_atmo_shield_generator"
+})
+
+Equipment.Register("hull.heavy_atmospheric_shielding", EquipType.New {
+	l10n_key="ATMOSPHERIC_SHIELDING_HEAVY",
+	price=900, purchasable=true, tech_level=5,
+	slot = { type="hull.atmo_shield", size=3 },
+	mass=5, volume=12, capabilities = { atmo_shield=19 },
+	icon_name="equip_atmo_shield_generator"
+})
+
+Equipment.Register("misc.hull_autorepair", EquipType.New {
+	l10n_key="HULL_AUTOREPAIR", slots="hull_autorepair",
+	price=16000, purchasable=true, tech_level="MILITARY",
+	slot = { type="hull.autorepair", size=4 },
+	mass=30, volume=40, capabilities={ hull_autorepair=1 },
+	icon_name="repairs"
+})
+
+--===============================================
+-- Thruster Mods
+--===============================================
+
+Equipment.Register("misc.thrusters_default", ThrusterType.New {
+	l10n_key="THRUSTERS_DEFAULT", slots="thruster",
+	price=120, purchasable=true, tech_level=2,
+	slot = { type="thruster", size=1 },
+	mass=0, volume=0, capabilities={ thruster_power=0 },
+	icon_name="equip_thrusters_basic"
+})
+
+Equipment.Register("misc.thrusters_basic", ThrusterType.New {
+	l10n_key="THRUSTERS_BASIC", slots="thruster",
+	price=250, purchasable=true, tech_level=5,
+	slot = { type="thruster", size=1 },
+	mass=0.1, volume=0.05, capabilities={ thruster_power=1 },
+	icon_name="equip_thrusters_basic"
+})
+
+Equipment.Register("misc.thrusters_medium", ThrusterType.New {
+	l10n_key="THRUSTERS_MEDIUM", slots="thruster",
+	price=560, purchasable=true, tech_level=8,
+	slot = { type="thruster", size=1 },
+	mass=0.05, volume=0.05, capabilities={ thruster_power=2 },
+	icon_name="equip_thrusters_medium"
+})
+
+Equipment.Register("misc.thrusters_best", ThrusterType.New {
+	l10n_key="THRUSTERS_BEST", slots="thruster",
+	price=14000, purchasable=true, tech_level="MILITARY",
+	slot = { type="thruster", size=1 },
+	mass=0, volume=0, capabilities={ thruster_power=3 },
+	icon_name="equip_thrusters_best"
+})
+
+--===============================================
+-- Scoops
+--===============================================
+
+Equipment.Register("misc.fuel_scoop", EquipType.New {
+	l10n_key="FUEL_SCOOP",
+	price=3500, purchasable=true, tech_level=4,
+	slot = { type="scoop.fuel", size=1, hardpoint=true },
```

Comment:
Need scoops of varying sizes and/or allow fitting smaller scoops.

Currently larger ships cannot fit any scoops.

Reply:
Please convert this to an issue so I can link it to https://github.com/orgs/pioneerspacesim/projects/2 as a Stage 2 task.